build-system/pr-check/validator-tests.js

```diff
@@ -26,6 +26,9 @@ const {runCiJob} = require('./ci-job');
 const jobName = 'validator-tests.js';
 
 function pushBuildWorkflow() {
+  timedExecOrDie('bazel test validator_test', {
```

Comment:
Looks like we are assuming that `bazel` is available on our CI machines. Not sure if that's indeed true. Might need to add an installation step here, similar to `Install Protobuf`.

https://github.com/ampproject/amphtml/blob/2bca7ebe2323a281dfe7f943fd59e98cbb9aa33b/.circleci/config.yml#L134-L145